modules/minimint.nix

```diff
@@ -0,0 +1,125 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  options.services.minimint = {
+      enable = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Enable Minimint,is a federated Chaumian e-cash mint backed 
+        by bitcoin with deposits and withdrawals that can occur on-chain
+        or via Lightning.
+      '';
+    }; 
+    address = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = "Address to listen for RPC connections.";
+    };
+    port = mkOption {
+      type = types.port;
+      default = 5001;
+      description = "Port to listen for RPC connections.";
+    };
```

Comment:
We will probably need this a second time for the p2p connections between federation members. But that might not be implemented yet, so probably ok for now.